Ghidra/Features/Base/src/main/java/ghidra/app/cmd/function/CreateFunctionCmd.java

```diff
@@ -281,7 +280,8 @@ private boolean createFunction(TaskMonitor monitor, String funcName,
 				if (!recreateFunction) {
 					long bodySize = functionContaining.getBody().getNumAddresses();
 					if (bodySize != 1) {
-						return false;
+						newFunc = functionContaining;
```

Comment:
Can you explain why changes to `CreateFunctionCmd.java` are needed?

Reply:
Unfortunately, due to the fact that the changes were made more than three years ago, I do not remember the details in this regard. Everything I know is written in the description of the comit. Unfortunately, now I no longer have a license for IDA to reproduce the behavior of the plugin to find out why I made such changes. Please excuse me for not being able to explain the changes made.